app/assets/stylesheets/dashboard.scss

```diff
@@ -257,45 +259,44 @@
   }
 }
 
-.charts-container {
-  max-width: 500px;
-  height: 250px;
-  margin: 0 auto;
-}
-
 .analytics-container {
   width: 1400px;
   margin: 0 auto;
 }
 
+.crayons-tabs--analytics {
+  justify-content: center;
+}
+
 .summary-stats {
+  text-align: center;
   display: flex;
-  flex-wrap: wrap;
-  width: 100%;
   justify-content: space-between;
 
-  .card {
-    flex: 1 0 150px;
+  &__col {
+    flex: 1;
```

Comment:
My only suggestion would be to extract `1` here into a variable to that it can be easily changed in the future on like `277` and `287` (or removed from both places if necessary):

```
$column-flex: 1

&__col {
  flex: $column-flex;
}
```